app/assets/stylesheets/modules.css.scss

```diff
@@ -31,6 +31,8 @@
     &.text { background-color:#F5F5EB; background-color:rgba(245, 245, 235, 0.95); }
     &.video { background-color:#EBF5EB; background-color:rgba(235, 245, 235, 0.95); }
     &.tutorial { background-color:#EBEBF5; background-color:rgba(235, 235, 245, 0.95); }
+    &.book { background-color:#EBF5F5; background-color:rgba(235, 245, 245, 0.95); }
```

Comment:
Color literals like `#EBF5F5` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Property `background-color` already defined on line 34<br>Color `#EBF5F5` should be written as `#ebf5f5`<br>Colon after property should be followed by one space